node/node.go

```diff
@@ -869,8 +874,10 @@ func NewNodeWithContext(ctx context.Context,
 	}
 
 	// Determine whether we should do fast sync. This must happen after the handshake, since the
-	// app may modify the validator set, specifying ourself as the only validator.
-	fastSync := config.FastSyncMode && !onlyValidatorIsUs(state, pubKey)
+	// app may modify the validator set, specifying ourselves as the only validator.
+	fastSync := config.FastSyncMode && !onlyValidatorIsUs(state,
+		pubKey) // the state certainly has the latest timeouts according
```

Comment:
[nit] moving this to a new line is less readable than it was before.

```suggestion
	fastSync := config.FastSyncMode && !onlyValidatorIsUs(state, pubKey) // the state certainly has the latest timeouts according
```

the code comment also seems incomplete: according to what?

Reply:
This comment wasn't addressed.